build-tools.Dockerfile

```diff
@@ -65,6 +69,10 @@ COPY --from=pgcopydb_builder /pgcopydb/lib/libpq.so.5 /pgcopydb/lib/libpq.so.5
 #
 # 'gdb' is included so that we get backtraces of core dumps produced in
 # regression tests
```

Comment:
The comment seems to be misplaced now

Reply:
thanks, fix in next commit and I will push it, just going to sync on next comment, so I could combine both fixes in one push :)